scripts/executors/src/start.ts

```diff
@@ -195,15 +200,22 @@ async function getSelectedTarget(
   targetsDescription: Record<string, string>,
   omitTargets: string[],
   createDescription: TargetDescriptionFn,
+  preselectTarget: PreselectTargetFn,
 ): Promise<string> {
   const projectConfig = projects[selectedProject];
   const projectTargets = projectConfig.data.targets ?? {};
+  const preselectedTarget = preselectTarget(projectConfig);
 
   const availableTargets = Object.keys(projectTargets)
     .filter(targetName => {
       return omitTargets.includes(targetName) ? false : true;
     })
+    .sort()
     .concat('help');
+  const preselectedTargetChoice =
+    preselectedTarget && availableTargets.indexOf(preselectedTarget) !== -1
```

Comment:
nit

```suggestion
    preselectedTarget && availableTargets.includes(preselectedTarget)
```

Reply:
using indexOf for consistency of the ternary